concrete/src/Database/Connection/ConnectionFactory.php

```diff
@@ -30,8 +30,11 @@ public function createConnection($config)
         $params = $config;
         $params['host'] = array_get($params, 'host', array_get($config, 'server'));
         $params['user'] = array_get($params, 'user', array_get($config, 'username'));
+        if (!isset($params['driverOptions'])) {
+            $params['driverOptions'] = array();
```

Comment:
This all looks good other than that we should be using PHP 5.4+ array syntax `[]`